vue3-apollo-quasar/src/components/RepoSubHeader/RepoSubHeader.vue

```diff
@@ -47,7 +47,10 @@
               <q-icon class="fa fa-caret-down"></q-icon>
             </span>
           </button>
-          <div class="dropdown_menu q-menu" v-if="refWatchMenu">
+          <div
+            class="dropdown_menu q-menu dropdown_menu--watch"
```

Comment:
Same here, would recommend we pick a convention and stick to it through out the codebase

Reply:
@hdJerry